ietf/api/urls.py

```diff
@@ -39,6 +39,8 @@
     url(r'^iesg/position', views_ballot.api_set_position),
     # Let Meetecho set session video URLs
     url(r'^meeting/session/video/url$', meeting_views.api_set_session_video_url),
+    # Let Meetecho tell us the name of its recordings
+    url(r'meeting/session/recordingname$', meeting_views.api_set_meetecho_recording_name),
```

Comment:
Need a `^` at the start.

Maybe make it `^meeting/session/recording-name` to better match `recording_name` in the model (and kebab-case that we've used in other URLs, like `doc/draft-aliases/` at line 27)

Reply:
will get the ^.
I can kebob case it, but I don't think we have consistency there.